lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -352,7 +375,7 @@ namespace Js
         func->GetFunctionProxy()->SetIsPublicLibraryCode();
         func->GetFunctionProxy()->EnsureDeserialized()->SetDisplayName(methodName->GetString(), methodName->GetLength(), 0);
 
-        DynamicObject* prototype = GetPrototypeFromName(JavascriptOperators::GetPropertyId(className, scriptContext), scriptContext);
+        DynamicObject* prototype = GetPrototypeFromName(JavascriptOperators::GetPropertyId(className, scriptContext), staticMethod, scriptContext);
```

Comment:
small nit: we have been trying to prefer `bool` to `BOOL`, and we turned off the warning about type mismatch for that conversion so it should always be implicit.